internal/services/controllers/workflows/queue.go

```diff
@@ -746,9 +774,7 @@ func (wc *WorkflowsControllerImpl) queueByGroupRoundRobin(ctx context.Context, t
 
 func (wc *WorkflowsControllerImpl) cancelWorkflowRun(ctx context.Context, tenantId, workflowRunId string) error {
 	// cancel all running step runs
-	runningStatus := dbsqlc.StepRunStatusRUNNING
 	stepRuns, err := wc.repo.StepRun().ListStepRuns(ctx, tenantId, &repository.ListStepRunsOpts{
```

Comment:
confirming: i suppose we're assuming downstream we're not canceling things in a finalized state?

Reply:
Discussed offline, we may cancel things in a finalized state, but we're intentionally leaving the behavior undefined for now, as it can end up in either of the two states.